student.rb

```diff
@@ -1,6 +1,8 @@
 require './person'
 # Student Class Inheriting From Person Class
 class Student < Person
+  attr_accessor :classroom
+
```

Comment:
- [ ] there should be a method that serves as the 'belongs-to' side (a student belongs to a classroom)